tests/test_assets.py

```diff
@@ -589,3 +594,134 @@ def test_security_dates_warning(self):
             for warning in w:
                 self.assertTrue(issubclass(warning.category,
                                            DeprecationWarning))
+
+    def test_root_symbol(self):
+        metadata = {
+            0: {
+                'symbol': 'OMH15',
+                'root_symbol': 'OM',
+                'asset_type': 'future',
+                'notice_date': '2014-01-20',
+                'expiration_date': '2014-02-20',
+                'contract_multiplier': 500
+            }
+        }
+
+        finder = AssetFinder(metadata=metadata)
+        omh15 = finder.retrieve_asset(0)
+        self.assertEqual('OM', omh15.root_symbol)
+
+    def test_lookup_future_in_chain(self):
+        metadata = {
+            2: {
+                'symbol': 'ADN15',
+                'root_symbol': 'AD',
+                'asset_type': 'future',
+                'expiration_date': pd.Timestamp('2015-06-15', tz='UTC')
+            },
+            1: {
+                'symbol': 'ADV15',
+                'root_symbol': 'AD',
+                'asset_type': 'future',
+                'expiration_date': pd.Timestamp('2015-09-14', tz='UTC')
+            },
+            0: {
+                'symbol': 'ADF16',
+                'root_symbol': 'AD',
+                'asset_type': 'future',
+                'expiration_date': pd.Timestamp('2015-12-14', tz='UTC')
+            },
+
+        }
+
+        finder = AssetFinder(metadata=metadata)
+        dt = pd.Timestamp('2015-06-19', tz='UTC')
+
+        primary = finder.lookup_future_in_chain('AD', dt, 1)
+        self.assertEqual(primary.sid, 1)
+
+        secondary = finder.lookup_future_in_chain('AD', dt, 2)
+        self.assertEqual(secondary.sid, 0)
+
+        self.assertIsNone(finder.lookup_future_in_chain('AD', dt, 0))
+        self.assertIsNone(finder.lookup_future_in_chain('AD', dt, -10))
+        self.assertIsNone(finder.lookup_future_in_chain('AD', dt, 10))
+        self.assertIsNone(finder.lookup_future_in_chain('CL', dt, 1))
```

Comment:
Can you add comments on what we are checking for with these assertions (640-649)? Alternatively, you can add an error message to the assertions that would double as a comment.